controllers/profiles/followers.js

```diff
@@ -54,7 +68,7 @@ module.exports = async (req, res) => {
         from
           "user_follow_user" AS "UserFollowUserCheckFollower" 
         where 
-          "UserFollowUserCheckFollower"."user_id_follower" = :id AND
+          "UserFollowUserCheckFollower"."user_id_follower" = :userId AND
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The replacement of `id` with `userId` is consistent with the changes made earlier in the code. This maintains consistency throughout the codebase.
```diff
- "UserFollowUserCheckFollower"."user_id_follower" = :id AND
+ "UserFollowUserCheckFollower"."user_id_follower" = :userId AND
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->